cpp2python.py

```diff
@@ -278,6 +295,92 @@ def process_line(line):
                 '\s*([^,]+)\s*,\s*' + \
                 'S[A-Z]+\s*\(\s*([\w\d]+)[^\)]+\)\s*\)\s*\)',
               '\\1.\\2.connect(\\3.\\4)', line)
+    
+
+    """ alter for statement
+    for(j = 1; j < c - 1; j+=1)
+                v
+    for j in range(1,c-1,1):
+        
+    """
+    regex = r"for\s?\(\s?(\s?(\w+)\b\s?=\s?(\d+|\w+)\b\s?)?;\s?((\w+)\s?(>|>=|<|<=|==|!=)?\s?((\w+|\d+)?\s?(\-|\+|\*|\/)?\s?(\w+|\d+)?))?\s?;\s?(\w+\s?(\+\+|\-\-|\+|\-)(\d+)?(\w+)?)?\s?\)"
```

Comment:
Wouldn't this reg exp replace both `for(j = 1; j < c - 1; j+=1)` and `for(j = 1; j >= c - 1; j-=1)` with `for j in range(1,c-1,1):`?
In this case hard-to-track bug might be introduced which is not visible for human who edits generated code.